dev-python/regex/regex-2019.12.20.recipe

```diff
@@ -0,0 +1,64 @@
+SUMMARY="All things regular expression: implemented in python"
+DESCRIPTION="Regular expression module, to replace re"
+HOMEPAGE="https://pypi.org/project/regex/"
+COPYRIGHT="2013-2019 Matthew Barnett"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://pypi.org/packages/source/r/regex/regex-$portVersion.tar.gz"
+CHECKSUM_SHA256="106e25a841921d8259dcef2a42786caae35bc750fb996f830065b3dfaa67b77e"
+SOURCE_DIR="regex-2019.12.20"
```

Comment:
Is SOURCE_DIR really needed?

Reply:
Yes, it failes without it at least onx86_64.

Reply:
The current value is the default value, si it shouldn't be needed.